CMakeLists.txt

```diff
@@ -482,8 +489,8 @@ set(libxdiff_SOURCES
 add_library(xdiff STATIC ${libxdiff_SOURCES})
 
 set(libvcs-svn_SOURCES
-vcs-svn/line_buffer.c vcs-svn/sliding_window.c vcs-svn/fast_export.c 
-vcs-svn/svndiff.c vcs-svn/svndump.c)
+	vcs-svn/line_buffer.c vcs-svn/sliding_window.c vcs-svn/fast_export.c 
+	vcs-svn/svndiff.c vcs-svn/svndump.c)
```

Comment:
This white-space change should be squashed into the commit that introduced the under-indented lines.